app/views/ProfileView/index.js

```diff
@@ -109,6 +110,7 @@ class ProfileView extends React.Component {
 		this.setState({
 			name,
 			username,
+			intialUsername: username,
```

Comment:
These `initialUsername` are not needed.

Reply:
Sorry, that was related to another issue, I made a mistake here. But now I have fixed it